src/ophyd_async/core/_signal.py

```diff
@@ -490,6 +490,75 @@ async def get_value():
         signal.clear_sub(q.put_nowait)
 
 
+async def observe_signals_values(
```

Comment:
After the above changes are made, can we share any code with `observe_value` above?